postgres/pgdump.go

```diff
@@ -585,6 +620,10 @@ func updateSchema(conv *internal.Conv, table string, cs []constraint, stmtType s
 			ct := conv.SrcSchema[table]
 			ct.ForeignKeys = append(ct.ForeignKeys, toForeignKeys(c)) // Append to previous foreign keys.
 			conv.SrcSchema[table] = ct
+		case nodes.CONSTR_UNIQUE:
+			ct := conv.SrcSchema[table]
+			ct.Indexes = append(ct.Indexes, schema.Index{Name: "", Unique: true, Keys: toSchemaKeys(conv, table, c.cols)})
+			conv.SrcSchema[table] = ct
```

Comment:
This is non-trivial code, and deserves a comment on the line of -

```
\\ Convert unique column constraint in postgres to a corresponding unique index in Spanner since Spanner doesn't support unique constraints on columns.
```

Reply:
added comment